src/main/java/org/springframework/integration/kafka/listener/ConcurrentMessageListenerDispatcher.java

```diff
@@ -120,7 +124,7 @@ private void initializeAndStartDispatching() {
 		for (int i = 0; i < consumers; i++) {
 			QueueingMessageListenerInvoker queueingMessageListenerInvoker =
 					new QueueingMessageListenerInvoker(queueSize, offsetManager, delegateListener, errorHandler,
-							taskExecutor);
+							taskExecutor, autoAckOnError);
```

Comment:
It's called `autoCommitOnError` in the `KafkaMessageListenerContainer` ... minor anyway.